.github/workflows/main.yml

```diff
@@ -23,20 +23,22 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+
     - uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
+
     - uses: actions/cache@v2
       with:
         path: ~/.cache/pip
         key: ${{ runner.os }}-pip-${{ hashFiles('requirements/*.txt') }}
         restore-keys: |
           ${{ runner.os }}-pip-
-    - name: Upgrade packaging tools
-      run: python -m pip install --upgrade pip setuptools virtualenv
+
     - name: Install dependencies
-      run: python -m pip install --upgrade tox
-    - name: Run tox targets for ${{ matrix.python-version }}
       run: |
-        ENV_PREFIX=$(tr -C -d "0-9" <<< "${{ matrix.python-version }}")
-        TOXENV=$(tox --listenvs | grep "^py$ENV_PREFIX" | tr '\n' ',') python -m tox
+        python -m pip install --upgrade pip setuptools wheel
+        python -m pip install --upgrade tox tox-py
+
+    - name: Run tox targets for ${{ matrix.python-version }}
+      run: tox --py current
```

Comment:
I guess we could also use `tox-gh-actions`? That's what most Jazzband projects use


e.g.
https://github.com/charettes/django-seal/blob/abb29508fc6faca633541dcef00ef3fa9f1b684e/.github/workflows/test.yml#L36-L43
https://github.com/jazzband/django-model-utils/blob/master/.github/workflows/test.yml

Reply:
Yeah if you want. I don't like having to redeclare which pyhons should run which envs, a bit redundant and another potential for failed maintenance. Given this package is getting obsolete I don't care that much about this PR though.